lib/src/browser.dart

```diff
@@ -89,11 +89,15 @@ class _Chrome extends Browser {
   static Version _getVersion(NavigatorProvider navigator) {
     Match match = new RegExp(r"Chrome/(\d+)\.(\d+)\.(\d+)\.(\d+)\s")
         .firstMatch(navigator.appVersion);
```

Comment:
If the issue is that the chromeless version number is formatted as x.x.x rather than x.x.x+x, could we just update this regex and the logic below to account for that instead of just silencing the error?